static/js/client.js

```diff
@@ -57,14 +57,15 @@ function cancelEditMessage() {
 }
 
 
-//REQUIRED FIELDS: id, status, name, user_id, ta_id, topic_name, ta_full_name
+//REQUIRED FIELDS: id, status, name, user_id, ta_id, topic_name, ta_full_name, question
 function buildTAEntry(entry) {
     var elt = $(entryHtml);
     elt.data("entryId", entry.id);
     elt.find(".id-input").val(entry.id);
     elt.find(".primary-content").html(
         "<div>" + entry.name + " (" + entry.user_id + ")</div>"
-      + "<div>" + entry.topic_name + "</div>"
+        + "<div>" + entry.topic_name + "</div>"
+        + "<div>" + entry.question + "</div>"
```

Comment:
Instead of having separate columns for the topic and question, why not format them together with the topic in [square brackets] at the start? Like this:
`"<div>[" + entry.topic_name + "] " + entry.question + "</div>"`
That way you don't have to worry about messing with the column layout.

Also, to avoid the huge line spacing for multiline questions, you'll need to add a class to the div and change the CSS rule on line 110 of styles.css so that it doesn't apply the tall line-height to the div.